### PR TITLE
[7.1] Validate stack modules' metricsets when xpack.enabled = true (#12386)

### PR DESCRIPTION
**Pull request by ycombinator**
Backports the following commits to 7.1:
 - Validate stack modules' metricsets when xpack.enabled = true  (#12386)

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -86,6 +86,25 @@ https://github.com/elastic/beats/compare/v7.1.1...7.1[Check the HEAD diff]
 
 *Metricbeat*
 
+- Add AWS SQS metricset. {pull}10684[10684] {issue}10053[10053]
```
Comment (cachedout) [inline CHANGELOG.next.asciidoc]: These look unrelated. Were these entries included in this backport errantly?

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: They were indeed 😞. Thanks for catching. I'll make a PR now to remove them.

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: Here's the PR: https://github.com/elastic/beats/pull/12473.
